src/main/java/com/savvato/tribeapp/controllers/dto/ReviewDecisionRequest.java

```diff
@@ -8,7 +8,7 @@ public class ReviewDecisionRequest {
   public Long reviewId;
 
   @Schema(example = "1")
-  public Long reviewerId;
+  public Long userId;
```

Comment:
What was the reason behind this change?

Reply:
we did this change as we were discussing to endpoint from the backend to make sure name match with the variable in the backend to their won't be any issue with call response.

Reply:
I kinda sorta recall that conversation.. Even though the userId variable will be coming from the frontend by default, the frontend should send us also a `reviewerId` parameter. It is the same value, but it's purpose is clearer. Revert this change, and I believe there is a related usage of this variable in another file, change that to `reviewerId` as well.

If I did say to change the backend, I apologize, I misspoke.